packages/date-picker/test/keyboard-input.test.js

```diff
@@ -87,49 +87,6 @@ describe('keyboard', () => {
     });
   });
 
-  describe('invalid date', () => {
-    it('should not select date on Enter if input invalid', async () => {
```

Comment:
This test is essentially the duplicate of the one added here, so it can be removed.

https://github.com/vaadin/web-components/blob/2b0859f55b105278acf722dc25f8f5a5a52bbcc8/packages/date-picker/test/validation.test.js#L205-L208